packages/gatsby-theme-bulmaio/gatsby-browser.js

```diff
@@ -0,0 +1,6 @@
+import Prism from 'prism-react-renderer/prism'
+
+(typeof global !== "undefined" ? global : window).Prism = Prism;
+
+require("prismjs/components/prism-kotlin");
+require("prismjs/components/prism-csharp");
```

Comment:
```suggestion
require("prismjs/components/prism-csharp");

// support c#
Prism.languages["C#"] = Prism.languages["c#"] = Prism.languages.dotnet;
```

Reply:
Because I'm freaking lazy and don't want to type `csharp`

Reply:
If it works, fine with me.

Reply:
@khalidabuhakmeh Did this still allow for `csharp` as well? If yes, let's accept the change :)

Reply:
Yes, you can think of `languages` like a dictionary. Prism looks at the `languages` object and indexes into it using the syntax at the end of the ticks. In this case I use `c#` or `C#` in addition to `csharp`